### ISSUE TITLE
How to change grid appearance in pyplot()?

### ISSUE THREAD

**Issue by PedroSapichu** [JuliaPlots/Plots.jl#704]
Hello all!,

I can't figure out why I always get straight lines instead of dashed lines for the grids with pyplot() backend. To the left I show the plot that's used as an example at juliaplots.github, and to the right is the plot I get with the same code (and any other plotting command). With other backends I get the normal dashed line that looks much better.

Thanks!

![grid](https://cloud.githubusercontent.com/assets/25947175/23324142/219ed8e2-faba-11e6-89f2-ba1674fbfb7b.png)

**Comment by jgbos** [comment]
I'll add that I'm trying to figure this out too.  It only happened recently and I'm not sure if it was `conda update` or `Pkg.update` that caused it.  For now I just initialize with `pyplot(grid=false)`.

**Comment by xiaoweiz** [comment]
Use the keyword arg `foreground_color_grid` and assign it to `lightgrey` may alleviate the issue, e.g. 
```julia
plot(x, y, foreground_color_grid=:lightgrey)
```
But I couldn't find a way to change the `linestyle` of the grid to be `dot`.

**Comment by jgbos** [comment]
There doesn't seem to be a keyword to set the grid style.  Here is what you can do.

Set defaults for grid by setting the following values in your `matplotlibrc` file:

```
### GRIDS
grid.color       :   d3d3d3    # grid color
grid.linestyle   :   :         # solid
grid.linewidth   :   0.5       # in points
grid.alpha       :   0.5       # transparency, between 0.0 and 1.0
```

I'm using matplotlib 2.0.  It appears the default `matplotlibrc` file has the grid lines being solid.

**Comment by xiaoweiz** [comment]
@jgbos I'm also using matplotlib 2.0 but its default appearance on my system has no grid.

**Comment by mkborregaard** [comment]
There is a PR to fix here https://github.com/JuliaPlots/Plots.jl/pull/721

**Comment by mkborregaard** [comment]
Closed by #721 but see the discussion there for a recommended enhancement.
